### DISCUSSION TITLE
Looking for the hardware file

### DISCUSSION THREAD

**Discussion by ChrisTietjen** [kachurovskiy/nanoels#226] [General]
I just purchased some PC boards for the H4.  I can't find the list of components anywhere.  Maybe I'm missing something simple but could someone point me to them.  Thanks.

Edit:  My mistake. I thought the H4 and the MegaEls were the same animal.  I'm all set for now.  Looking forward to building the project.

**Comment by kachurovskiy** [comment]
Hi @ChrisTietjen, components are defined in BOM files in https://github.com/kachurovskiy/nanoels/tree/main/h4/pcbs - they are human-readable but are usually handled by the PCB producer software.
